torch/distributed/rpc/api.py

```diff
@@ -424,8 +426,8 @@ def rpc_sync(to, func, args=None, kwargs=None):
 
     Arguments:
         to (str or WorkerInfo): id or name of the destination worker.
-        func (callable): any callable function. builtin functions (like
-                         :meth:`torch.add`) can be sent over RPC more efficiently.
+        func (callable): any callable function. builtin or annotated TorchScript
```

Comment:
Still feel it's not clear. 

Python function(callable to be precise, which includes lambda too), seems not mentioned from the comment.

```
func (callable): any callable function. builtin or annotated TorchScript
```

to

```
func (callable): any function. Python callable, PyTorch built-in function or annotated TorchScript.
```